Sample Data/SAPBTPAuditLog_Schema.csv

```diff
@@ -0,0 +1,14 @@
+﻿ColumnName,ColumnOrdinal,DataType,ColumnType
```

Comment:
This file is not required. SAPBTPAuditLog_CL is enough to give the idea of the schema.

Reply:
I was following the guide [here](https://github.com/Azure/Azure-Sentinel/blob/master/Sample%20Data/README.md). Does it need to be updated as the wording suggests the file is needed?